legacy/app/data/common/post-edit-create/location.directive.js

```diff
@@ -187,11 +187,15 @@ function PostLocationDirective($document, $http, L, Geocoding, Maps, _, Notify,
             element[0].querySelector('#searchbar-results').classList.remove('active');
         }
 
+        function test1() {
+            console.log('chege test');
+        }
+
         function handleActiveSearch(event) {
-            var del = event.keyCode === 8 || event.keyCode === 46;
-            var letter = event.keyCode > 47 && event.keyCode < 58;
-            var number = event.keyCode > 64 && event.keyCode < 91;
-            if (del || letter || number) {
+            // var del = event.keyCode === 8 || event.keyCode === 46;
+            // var letter = event.keyCode > 47 && event.keyCode < 58;
+            // var number = event.keyCode > 64 && event.keyCode < 91;
+            if (event.keyCode === 32) {
```

Comment:
A cool attempt at solving this @chegejohn159 

I have the following comments:
- Even though I see that you added this `<p>Press Space Bar After Typing Location</p>` in the html to direct the user to use the space bar, the problem is that (from experience) many users naturally expect to press enter key in this case based on the norm/convention. Some won't even read what you put there and just assume the feature doesn't work.
- About enter key and reload: It is the default behavior for enter key to reload. It is also a normal thing to use event.preventDefault() to prevent that reload. There is nothing wrong with doing that.

So I'd suggest removing the html instruction you added and use the enter key based on norm or what a user would naturally expect.

Here's what your handleActiveSearch function can look like:

 ```
function handleActiveSearch(event) {
            if (event.keyCode === 13) {
                event.preventDefault();
                $scope.processing = true;
                if ($scope.searchTimeout) {
                    clearTimeout($scope.searchTimeout);
                }
                $scope.searchTimeout = setTimeout($scope.searchLocation, 250);
            }
        }
```